CustomizableActionSheet/CustomizableActionSheet.swift

```diff
@@ -93,7 +92,8 @@ public class CustomizableActionSheet: NSObject {
   private var closeBlock: (() -> Void)?
 
   // MARK: - Public properties
-
+  
+  public var cornerRadius: CGFloat = 4
```

Comment:
I think it's a good idea.
But, its name is not easy to understand.

This variable means the default value of corner radius.
So, I think better to rename with `defaultConerRadius`.